basic-features.md

```diff
@@ -10,192 +10,197 @@ aliases: ['/docs-cn/dev/basic-features/']
-| [Titan 插件](/storage-engine/titan-overview.md)                        |  Y   |  Y   |  Y    |  Y    |  Y    |    Y     |    Y     |    Y     |    Y     |    Y     |    Y     |
-| [不可见索引](/sql-statements/sql-statement-add-index.md)               |  Y   |  Y   |  Y    |  Y    |  Y    |    Y     |    Y     |    Y     |    Y     |    Y     |    N     |
-| [复合主键](/constraints.md#主键约束)                                   |  Y   |  Y   |  Y    |  Y    |  Y    |    Y     |    Y     |    Y     |    Y     |    Y     |    Y     |
-| [唯一约束](/constraints.md#唯一约束)                                   |  Y   |  Y   |  Y   |  Y    |  Y    |    Y     |    Y     |    Y     |    Y     |    Y     |    Y     |
-| [整型主键上的聚簇索引](/constraints.md)                                |  Y   |  Y   |  Y    |  Y    |  Y    |    Y     |    Y     |    Y     |    Y     |    Y     |    Y     |
-| [复合或非整型主键上的聚簇索引](/constraints.md)                        |  Y   |  Y   |  Y    |  Y    |   Y   |    Y     |    Y     |    Y     |    Y     |    Y     |    N     |
+| 索引和约束                                                             | 6.5 | 6.4 | 6.3 | 6.2 | 6.1 |  6.0   |   5.4    |   5.3    |   5.2    |   5.1    |   5.0    |   4.0    |
+| ---------------------------------------------------------------------- | --- | --- | --- | --- | --- | --- | :------: | :------: | :------: | :------: | :------: | :------: |
+| [表达式索引](/sql-statements/sql-statement-create-index.md#表达式索引) [^2] | Y | 实验特性 | 实验特性 | 实验特性  | 实验特性  | 实验特性  | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 |
+| [列式存储 (TiFlash)](/tiflash/tiflash-overview.md)                     |  Y   |  Y   |  Y   |  Y   |  Y    |  Y    |    Y     |    Y     |    Y     |    Y     |    Y     |    Y     |
+| [RocksDB 引擎](/storage-engine/rocksdb-overview.md)                    |  Y   |  Y   |  Y   |  Y    |  Y    |  Y    |    Y     |    Y     |    Y     |    Y     |    Y     |    Y     |
+| [Titan 插件](/storage-engine/titan-overview.md)                        |  Y   |  Y   |  Y   |  Y    |  Y    |  Y    |    Y     |    Y     |    Y     |    Y     |    Y     |    Y     |
+| [不可见索引](/sql-statements/sql-statement-add-index.md)               |  Y   |  Y   |  Y   |  Y    |  Y    |  Y    |    Y     |    Y     |    Y     |    Y     |    Y     |    N     |
+| [复合主键](/constraints.md#主键约束)                                   |  Y   |  Y   |  Y   |  Y    |  Y    |  Y    |    Y     |    Y     |    Y     |    Y     |    Y     |    Y     |
+| [唯一约束](/constraints.md#唯一约束)                                   |  Y   |  Y   |  Y   |  Y   |  Y    |  Y    |    Y     |    Y     |    Y     |    Y     |    Y     |    Y     |
+| [整型主键上的聚簇索引](/constraints.md)                                |  Y   |  Y   |  Y   |  Y    |  Y    |  Y    |    Y     |    Y     |    Y     |    Y     |    Y     |    Y     |
+| [复合或非整型主键上的聚簇索引](/constraints.md)                        |  Y   |  Y   |  Y   |  Y    |  Y    |   Y   |    Y     |    Y     |    Y     |    Y     |    Y     |    N     |
 
 ## SQL 语句
```

Comment:
@SunRunAway Do you want to add TTL to this document?

Reply:
It was already added in Line L96

| [TTL (Time to Live)](/time-to-live.md) | Experimental | N | N | N | N | N | N | N | N | N | N | N |